fec.config.js

```diff
@@ -1,8 +1,9 @@
 const path = require('path');
+const dependencies = require('./package.json').dependencies;
 
 module.exports = {
   appUrl: ['/staging/starter'],
-  sassPrefix: '.widgetLayout',
+  sassPrefix: process.env.CONFIG_PORT || process.env.PROXY ? '.frontendStarterApp' : '.widgetLayout',
```

Comment:
No need to add these as it breaks the deployment.
```suggestion
  sassPrefix: '.widgetLayout, .landing',
```